### ISSUE TITLE
Delete from database if both participant deleted conversation

### ISSUE THREAD

**Issue by dahnmills** [mailboxer/mailboxer#456]
Hi,

I am trying to delete a conversation with MailBoxer gem. There's nothing included in the gem itself, but I think it's possible to do something like:

If both participants deleted the conversation, delete it from the database.

I started with something like this in my function "empty_trash", so like that, every time a participant click on the link "Empty Trash", it will check if the conversation has been deleted by both or not. If not, nothing happens, if yes, it will delete the conversation from the DB.

`
def empty_trash

    @conversations = current_user.mailbox.conversations

        @conversations.each do |conversation|

            conversation.receipts_for(current_user).update_all(deleted: true)

        end

    @delete_conversation = Mailboxer::Receipt.select(:notification_id,:deleted).
                                         group(:notification_id,:deleted).
                                         having("count(*) > 1")

    @delete_conversation.destroy_all

    redirect_to :back

end
`

So basically, what I am trying to do, is to group all conversation by :notification_id where deleted: true and then delete the matching result.

If someone got an idea...!
Thanks!
